### ISSUE TITLE
Long wait time while roadmap is loading

### ISSUE THREAD

**Issue by skeeJay** [newjersey/business.nj.gov#210]
BLOCKED ON #213 

The user can be staring at the grey box with multiple "Loading…" for a long time, which can look broken. I'm not sure if there is a way to speed up this lookup, or we might otherwise want some other kind of animated progress indicator to make sure the user knows progress is coming.

Activities include:

- [ ]

**Comment by aloverso** [comment]
@skeeJay @guy-nj can definitely do something/anything here, let me know what kind of design you'd like

**Comment by skeeJay** [comment]
@aloverso Will we see this kind of load time in production?

**Comment by aloverso** [comment]
I was just about to say - I think this is being caused by the license search update, which I just added a chore for us to fix - #213 

So once we make this change, the long load time will be what you see _after_ that initial long load (since the long-load will happen once every hour or more)

To test what the load time _will actually_ be after we fix this, try this:
- log in for the first time after 1hr, see the long load time (7ish seconds)
- log out to clear your cache
- log back in again. should be a very short load time now, under 1 second

**Comment by skeeJay** [comment]
Okay. Yeah, I'm sure we'll continue to find ways to optimize things.

**Comment by skeeJay** [comment]
Looks improved to me. We'll create a new ticket if we see anything, but closing this.
